src/driver.rs

```diff
@@ -11,7 +11,9 @@ use query::Query;
 /// exist in the file system, and then return an instance initialized with the
 /// `connect` static trait function.
 pub fn get_driver<D: Driver>(url: &str) -> D {
-  // TODO: implement
+  // TODO: implement.
+  // Used to quiet warnings.
```

Comment:
Yup. The default is to be local. I had to put a `!` for it to apply to the whole module.